shared/src/main/scala/amf/core/model/document/AmfProcessingData.scala

```diff
@@ -0,0 +1,35 @@
+package amf.core.model.document
+
+import amf.core.metamodel.{Field, Obj}
+import amf.core.metamodel.document.AmfProcessingDataModel
+import amf.core.metamodel.document.AmfProcessingDataModel.{
+  GraphDependencies,
+  ModelVersion,
+  SourceVendor,
+  Transformations
+}
+import amf.core.model.StrField
+import amf.core.model.domain.{AmfObject, AmfScalar}
+import amf.core.parser.{Annotations, Fields}
+
+case class AmfProcessingData(fields: Fields, annotations: Annotations) extends AmfObject {
+
+  def transformations: Seq[StrField]   = fields.field(Transformations)
```

Comment:
At first sight it wasn't clear if this stored the transformation pipeline or the steps applied. Maybe rename to transformationSteps?